assemblyline_v4_service/common/ocr.py

```diff
@@ -29,6 +29,12 @@
         "enable macro",
         "enable content",
         "enable editing",
+        # https://github.com/CAPESandbox/community/blob/815e21980f4b234cf84e78749447f262af2beef9/modules/signatures/office_macro.py
+        "bank account",
```

Comment:
I question whether "bank account" should be grouped with the macro indicators. Would it be better to group it with the ransomware strings?

Reply:
addressed